core/execution.go

```diff
@@ -31,6 +32,9 @@ var (
 
 	maxCodeSize            = 24576
 	errMaxCodeSizeExceeded = fmt.Errorf("Max Code Size exceeded (%d)", maxCodeSize)
+
+	errCodeStoreOutOfGas        = errors.New("contract creation code storage out of gas")
+	errContractAddressCollision = errors.New("contract address collision")
```

Comment:
variable isn't used and references the unimplemented EIP, remove this and I'm good with merge